tests/pytorch_pfn_extras_tests/training_tests/extensions_tests/test_snapshot.py

```diff
@@ -384,3 +390,38 @@ def get_epoch_indices():
 
     epoch_indices = get_epoch_indices()
     assert len(epoch_indices) == 0
+
+
+def test_snapshot_autoload_initialize_without_writer(path):
+    snapshot_filename = "snapshot_file"
+    trainer = get_trainer(out_dir=path)
+    trainer.models["main"]._state_dict = {"value": 0}
+
+    snapshot = extensions.snapshot(filename=snapshot_filename)
+    snapshot(trainer)
+    assert os.path.isfile(f"{path}/{snapshot_filename}")
```

Comment:
Concat path with `/` string is rather unsafe. You might want to use `os.path.join()` instead?

Reply:
I agree. I fixed the code.